create_topics_request.go

```diff
@@ -83,6 +83,10 @@ func (r *CreateTopicsRequest) headerVersion() int16 {
 	return 1
 }
 
+func (c *CreateTopicsRequest) isValidVersion() bool {
+	return c.Version >= 0 && c.Version <= 2
```

Comment:
Version 3 seems to only be a server behaviour change?

```suggestion
	return c.Version >= 0 && c.Version <= 3
```

Reply:
Actually I think version 4 only changes semantics not fields and we would support it?